muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/Snackbar.kt

```diff
@@ -64,7 +64,7 @@ var MSnackbarProps.transitionDuration by TransitionDurationDelegate()
 
 
 fun RBuilder.mSnackbar(
-        message: ReactElement,
+        message: ReactElement?,
```

Comment:
I am wondering if it would be better to leave this non-nullable and add a new function without the message parameter (or swap the order of the params, but not sure if many people have used mSnackbar already)... mainly so we don't have to use a null when calling it...

So fun Rbuilder.mSnackbar (open: Boolean... etc)